packages/frontend/src/common/types/commonTypes.ts

```diff
@@ -2,11 +2,9 @@ export interface BaseAction {
   type: string;
 }
 
-export type UserRole = "moderator" | "participant";
-
 export interface User {
   id: string;
-  role: UserRole;
+  isModerator: boolean;
```

Comment:
I'd rather keep it as an enum. It might be YAGNI but we're way more flexible when it comes to adding new roles.